src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -216,20 +216,43 @@ public void hearingResponseUpdateRecover(UpdateCaseException exception, HearingW
         throw new ExhaustedRetryException("Cancellation request Response received, rethrowing exception", exception);
     }
 
-    private HearingWrapper createWrapper(HearingRequest hearingRequest) throws GetCaseException, UnhandleableHearingStateException {
+    private HearingWrapper createWrapper(HearingRequest hearingRequest) throws GetCaseException,
+        UnhandleableHearingStateException {
         if (isNull(hearingRequest.getHearingState())) {
             UnhandleableHearingStateException err = new UnhandleableHearingStateException();
             log.error(err.getMessage(), err);
             throw err;
         }
 
-        List<CancellationReason> cancellationReasons = nonNull(hearingRequest.getCancellationReason())
-            ? List.of(hearingRequest.getCancellationReason())
-            : null;
+        List<CancellationReason> cancellationReasons = null;
+
+        if (hearingRequest.getCancellationReason() != null) {
+            cancellationReasons = List.of(hearingRequest.getCancellationReason());
+        }
+
         return HearingWrapper.builder()
                 .caseData(ccdCaseService.getCaseDetails(hearingRequest.getCcdCaseId()).getData())
                 .state(hearingRequest.getHearingState())
                 .cancellationReasons(cancellationReasons)
                 .build();
     }
+
+    private void setDefaultOverrideFields(SscsCaseData caseData,
+                                          HearingWrapper wrapper,
+                                          ReferenceDataServiceHolder referenceDataServiceHolder)
+        throws InvalidMappingException, UpdateCaseException {
+        try {
+            OverridesMapping.setDefaultOverrideFields(wrapper, referenceDataServiceHolder);
+        } catch (ListingException ex) {
+            ccdCaseService.updateCaseData(
+                caseData,
+                LISTING_ERROR,
+                ListingException.SUMMARY,
+                ex.getMessage());
+
+            log.debug("Missing listing requirements found! State is now {}.", State.LISTING_ERROR);
```

Comment:
Dont Catch this here, you want to catch this higher up

Reply:
done